styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
+
+.toolbar {
+  min-height: 4rem;
+  display: flex;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+}
+
+.toolbar-items {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.bar-icon {
+  zoom: 1.5;
+}
+
+.bar-icon:hover {
+  zoom: 1.8;
+  transition: ease-out;
+  box-shadow: 1px 1px 5px gray;
+}
+
+#logo-text {
+  line-height: 2px;
+  font-weight: 500;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: url("/headline_bg.svg");
+  background-repeat: no-repeat;
+  min-height: 40vh;
+}
+
+.headline-section > h1 {
+  color: #7d73fc;
+  font-weight: 700;
```

Comment:
- [ ] As mentioned previously, even though you are setting the `font-weight: 700;` it's currently not working. Your font import (HTML file, line 20), there are only 100 and 500 weights. You should consider changing it to match the original design better.

Reply:
`font-weight: 700` added successfully